server/issue.go

```diff
@@ -120,6 +120,10 @@ func httpAPICreateIssue(ji Instance, w http.ResponseWriter, r *http.Request) (in
 		return http.StatusOK, nil
 	}
 
+	if issue.Fields.Reporter == nil && ji.GetType() == JIRATypeServer {
```

Comment:
Why not both cloud and server?

Reply:
The cloud (mmtest) error-ed out when Reporter was set, I lost the copy-pasted record of it, but basically was not supposed to be set. I tested on both Serveer 7 and 8, worked on both.

Reply:
Is there harm in making this consistent across Cloud and Server, instead of special casing it? Or are we certain that this doesn't affect Server and thus don't want to make any changes to it?

Reply:
Setting the Reporter field seems to be erroring out on the Cloud, it appears tto want to use the session user. I kept the "server-only" logic.